ops/terraform/services/migrator/deploy.groovy

```diff
@@ -90,10 +105,13 @@ String monitorMigrator(Map args = [:]) {
         // 2. if the message body contains a non "0/0" (running) value, return it
         for (msg in messages) {
             migratorStatus = msg.body.status
+            println "Migrator schema version is at ${msg.body.schema_version}"
+            schemaVersion = msg.body.schema_version
             printMigratorMessage(msg)
             awsSqs.deleteMessage(msg.receipt, sqsQueueUrl)
-            if (migratorStatus != '0/0') {
-                return migratorStatus
+            if (migratorStatus =='0') {
```

Comment:
Why did we switch the logic from `!== '0/0'1 to `== '0'`? Does the comment above need to be updated? Also - need a space after the `==` :)

Reply:
only concerned with the status of 0 now. but updated the comment and the space after

Reply:
I think this needs to be reviewed, @cbrunefearless. `'0/0'` should indicate that the systemd unit is still running, so if we get anything other than `'0/0'` that signals that the migrator has exited. It's possible that some magical thinking led us to update this value.

Reply:
yea 0/0 is still running, 0 means the migrator has sent over the latest schema version and is finished..

Reply:
> yea 0/0 is still running, 0 means the migrator has sent over the latest schema version and is finished..

Unfortunately, this check excludes migrator failure scenarios.